pages/draw.md

```diff
@@ -262,6 +262,37 @@ poi-icons:
             sprite_default: generic
 ```
 
+####`collide`
+Optional _boolean_. Defaults to `true`.
```

Comment:
Lets discuss rather.

Should a label/icon not be part of the collision check when marked `collide: false`?

Currently in native code base, )https://github.com/tangrams/tangram-es/blob/master/core/src/labels/label.cpp#L139 and https://github.com/tangrams/tangram-es/blob/master/core/src/labels/labels.cpp#L70), it seems a label/icon having `collide: false` will not take part in the collision check at all.

So to give an example, what should be the behaviour, when a label (with collide: true) collides with an icon (with collide: false)?
I will argue that in such a case, label should **not** be drawn as it is in collision with another icon, and icon wins the collision because of the `collide: false` property.

We need to clarify this behaviour.


@karimnaaji thoughts?